### PR TITLE
hero section 

### PR DESCRIPTION
**Pull request by barakadanny**
In this pull request I just created the home section (hero)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    </header>
+
+    <main class="main">
+        <section class="hero" id="hero">
+            <!-- background graphics -->
+            <img class="graph graph-1" src="img/graph_1.png" alt="graph-1">
+            <img class="graph graph-2" src="img/graph_2.png" alt="graph-2">
+            <img class="graph graph-3" src="img/graph_3.png" alt="graph-3">
+            <img class="graph graph-4" src="img/graph_4.png" alt="graph-4">
+            <img class="graph graph-5" src="img/graph_5.png" alt="graph-5">
```
Comment (1GORDON) [inline index.html:36]: Great Job so far @barakadanny.🤝. 

- [ ] Try to follow the Figma design provided in the project requirements
- Figma Design
Figma design
- Your design
Your design

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @barakadanny  

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Please add the correct linter files [Linter files](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [x] Please add the readme file and customise it accordingly [Readme file](https://github.com/microverseinc/readme-template/blob/master/README.md)

### _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by barakadanny** [comment]
note: the official readme file is on the master branch

**Comment by barakadanny** [comment]
note: The official readme file is on the master branch

**Review by aboongm — APPROVED** [review]
Hi @barakadanny , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

request_changes_1

## Highlights

## Optional suggestions

[OPTIONAL] Please update the following: 
- In the README.md the link for issues still does not work. 
- The MIT license is also not yet personalize.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**